From: Ivan Molodetskikh <yalterz@gmail.com>
Date: Fri, 28 Jan 2022 16:31:50 +0300
Subject: keyboard: Remove 01-screenshot.xml

Move this to gnome-shell where it belongs. See
https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/2107

(cherry picked from commit 3ecc15106131cc95b5a96dfe96721d3f44e0f830)
---
 panels/keyboard/01-screenshot.xml.in | 29 -----------------------------
 panels/keyboard/meson.build          |  1 -
 po/POTFILES.in                       |  1 -
 3 files changed, 31 deletions(-)
 delete mode 100644 panels/keyboard/01-screenshot.xml.in

diff --git a/panels/keyboard/01-screenshot.xml.in b/panels/keyboard/01-screenshot.xml.in
deleted file mode 100644
index 002bd38..0000000
--- a/panels/keyboard/01-screenshot.xml.in
+++ /dev/null
@@ -1,29 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" ?>
-<KeyListEntries group="system" schema="org.gnome.settings-daemon.plugins.media-keys" name="Screenshots">
-
-	<!-- translators: $PICTURES will be replaced by the name of the XDG Pictures directory -->
-	<KeyListEntry name="screenshot"
-		description="Save a screenshot to $PICTURES"/>
-
-	<!-- translators: $PICTURES will be replaced by the name of the XDG Pictures directory -->
-	<KeyListEntry name="window-screenshot"
-		description="Save a screenshot of a window to $PICTURES"/>
-
-	<!-- translators: $PICTURES will be replaced by the name of the XDG Pictures directory -->
-	<KeyListEntry name="area-screenshot"
-                      description="Save a screenshot of an area to $PICTURES"/>
-
-	<KeyListEntry name="screenshot-clip"
-                      description="Copy a screenshot to clipboard"/>
-
-	<KeyListEntry name="window-screenshot-clip"
-                      description="Copy a screenshot of a window to clipboard"/>
-
-	<KeyListEntry name="area-screenshot-clip"
-                      description="Copy a screenshot of an area to clipboard"/>
-
-	<KeyListEntry name="screencast"
-		description="Record a short screencast"/>
-
-</KeyListEntries>
-
diff --git a/panels/keyboard/meson.build b/panels/keyboard/meson.build
index ae71c94..6773902 100644
--- a/panels/keyboard/meson.build
+++ b/panels/keyboard/meson.build
@@ -37,7 +37,6 @@ xml_files = [
   '00-multimedia.xml',
   '01-input-sources.xml',
   '01-launchers.xml',
-  '01-screenshot.xml',
   '01-system.xml',
   '50-accessibility.xml'
 ]
diff --git a/po/POTFILES.in b/po/POTFILES.in
index b9d65b7..98b8848 100644
--- a/po/POTFILES.in
+++ b/po/POTFILES.in
@@ -54,7 +54,6 @@ panels/info-overview/gnome-info-overview-panel.desktop.in.in
 panels/keyboard/00-multimedia.xml.in
 panels/keyboard/01-input-sources.xml.in
 panels/keyboard/01-launchers.xml.in
-panels/keyboard/01-screenshot.xml.in
 panels/keyboard/01-system.xml.in
 panels/keyboard/50-accessibility.xml.in
 panels/keyboard/cc-input-chooser.c
